### PR TITLE
Feat/abdullahi/product page

### PR DESCRIPTION
**Pull request by S-Abdullahi**
image
image
image
image

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -63,6 +63,19 @@
   .container {
     @apply px-[3rem] md:px-["4rem"] lg:px-["6rem"] xl:px-["8rem"] mx-auto;
   }
+
```
Comment (pleasantvik) [inline src/styles/globals.css:66]: if this classname are not used across multiple component, remove them and do the styling per component

File: src/components/products/Features.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+const Features = () => {
+  return (
+    <div className="bg-[#EFFBFF] px-10 lg:px-[103px] nxl:pt-[131px] pt-20 pb-10 nxl:pb-[101px]">
+      {/* feature one */}
+      <div className="section-layout mb-10 lg:mb-0">
+        <div className="order-2 lg:order-1 flex justify-center">
+          <img src="/asset/images/FundWallet.svg" alt="fund wallet" />
```
Comment (pleasantvik) [inline src/components/products/Features.tsx]: Use Image component from next/image

File: src/components/products/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { BsArrowUpRight } from "react-icons/bs";
+
+const Hero = () => {
+  return (
+    <div>
+      <div className="lg:h-[658px] grid grid-cols-1 lg:grid-cols-2 items-center gap-4 px-10 md:px-[103px]">
+        <div className="absolute top-0 right-0 -z-10">
+          <img src="/asset/images/pattern3.svg" alt="pattern 3" />
```
Comment (pleasantvik) [inline src/components/products/Hero.tsx]: Use Image component from next/image

File: src/components/products/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          <p className="text-2xl mb-10 leading-8 nxl:w-[542px]">
+            Easy and secure way to manage transactions from your Business and
+            Personal platform.
+          </p>
+          <button className="flex items-center gap-2 bg-[#243AC0] text-white rounded-lg py-4 px-6">
+            Get Started <BsArrowUpRight />
+          </button>
+        </div>
+        <div className="flex justify-center">
+          <img src="/asset/images/Segurawallet.svg" alt="Segura Wallet" />
```
Comment (pleasantvik) [inline src/components/products/Hero.tsx]: Use Image component from next/image

File: src/components/products/Hero.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+const Hero = () => {
+  return (
+    <div>
+      <div className="lg:h-[658px] grid grid-cols-1 lg:grid-cols-2 items-center gap-4 px-10 md:px-[103px]">
+        <div className="absolute top-0 right-0 -z-10">
+          <img src="/asset/images/pattern3.svg" alt="pattern 3" />
+        </div>
+        <div className="absolute top-0 left-0 -z-10">
+          <img src="/asset/images/Pattern4.svg" alt="pattern 4" />
```
Comment (pleasantvik) [inline src/components/products/Hero.tsx]: Use Image component from next/image

File: src/components/products/Features.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+          <p className="section-description mb-10 lg:mb-0">
+            Security: SeguraWallet employs high-level security measures to
+            ensure the safety of its clients and customers' funds. The service
+            uses advanced encryption and secure storage methods to protect
+            sensitive data
+          </p>
+        </div>
+        <div className="flex justify-center">
+          <img src="/asset/images/Segurawallet2.svg" alt="" />
```
Comment (pleasantvik) [inline src/components/products/Features.tsx]: Use Image component from next/image

File: tailwind.config.js
Comment (pleasantvik) [inline tailwind.config.js:1]: Dont modify this file

File: src/styles/globals.css (hunk 1 above)
Comment (pleasantvik) [inline src/styles/globals.css:66]: if this classname are not used across multiple component, remove them and do the styling per component

Comment (S-Abdullahi) [reply src/styles/globals.css:66]: I was not the one that created that particular class

**Review by pleasantvik — APPROVED** [review]

**Pull request merged**